From c1b802eeb0307db3e90bd2fca702256652331758 Mon Sep 17 00:00:00 2001
From: Kirill Smelkov <kirr@mns.spb.ru>
Date: Wed, 10 Nov 2010 20:02:19 +0300
Subject: [PATCH 2/4] antiword: Tweak Makefile.Linux for msysGit

It is easier to start from Makefile.Linux because even if there is
Makefile.cygming as we'll see in later patches, it does not serve our purposes
well - we'll still need to change installed location and runtime path for
resources...

Also we don't need kantiword, and /usr/local/bin/ -> /usr/bin/

Signed-off-by: Kirill Smelkov <kirr@mns.spb.ru>
---
 Makefile.Linux |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.Linux b/Makefile.Linux
index eeb2d15..65af1df 100644
--- a/Makefile.Linux
+++ b/Makefile.Linux
@@ -30,13 +30,13 @@ OBJS	=\
 	wordlib.o wordmac.o wordole.o wordwin.o xmalloc.o xml.o
 
 PROGS =\
-	antiword\
-	kantiword
+	antiword.exe
+#	kantiword
 
 LOCAL_INSTALL_DIR = $(HOME)/bin
 LOCAL_RESOURCES_DIR = $(HOME)/.antiword
 
-GLOBAL_INSTALL_DIR = /usr/local/bin
+GLOBAL_INSTALL_DIR = /usr/bin
 GLOBAL_RESOURCES_DIR = /usr/share/antiword
 
 all:		$(PROGS)
@@ -71,7 +71,7 @@ clean:
 	rm -f $(OBJS)
 	rm -f $(PROGS)
 
-antiword:	$(OBJS)
+antiword.exe:	$(OBJS)
 	@rm -f $@
 	$(LD) $(LDFLAGS) $(OBJS) $(LDLIBS) -o $@
 	@chmod 750 $@
-- 
1.7.3.2.245.g03276

